Makefile

```diff
@@ -31,13 +31,13 @@ ginkgo:
 	unlink vendor/src
 
 functional: ginkgo
-	./ginkgo functional/ -- -runtime ${CC_RUNTIME} -timeout ${TIMEOUT}
+	./ginkgo -v functional/ -- -runtime ${CC_RUNTIME} -timeout ${TIMEOUT}
```

Comment:
nit - obviously you also snuck the `-v` in here as well - I presume that is verbose, and not version output ?

Reply:
hehe - I missed to split the commit, I partially  modify the makefile to see the commands that are being running ( `-v`) and found really useful to only tests I want `-focus`. I think are useful let me split the commit to have that merged in this PR.

Reply:
@grahamwhaley  now is explained why is added -v and -focus